samples/dashboards/demo/accounting/demo.css

```diff
@@ -0,0 +1,194 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/dashboards/css/dashboards.css");
+@import url("https://code.highcharts.com/datagrid/css/datagrid.css");
+
+/* Dashboards */
+
+.highcharts-dashboards-component-title {
```

Comment:
Could be even smaller:

font-size: 0.9rm
margin-bottom: 8px
color: var(--highcharts-neutral-color-100);